### PR TITLE
Added map on the main screen

### PR DESCRIPTION
**Pull request by marcosjvivar**
Added map on the main screen using an IBOutlets MGLMapView instead of doing it by code.

### CODE REVIEW

File: MyBus/MyBus/AppDelegate.swift [hunk 1]
```diff
@@ -22,6 +22,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool
     {
         Fabric.with([Crashlytics.self, MGLAccountManager.self])
+        
+        //MGLAccountManager.setAccessToken("pk.eyJ1Ijoibm9zb3VsODgiLCJhIjoiY2lteGt2dHhsMDNrNXZxbHU0M29mcHZnZiJ9.MMbmK9GfcdhpDw2siu0wuA")
```
Comment (lisandro) [inline MyBus/MyBus/AppDelegate.swift:26]: Quick question @mvivardevspark . Will we use this way to set Mapbox's token in the future?

Comment (marcosjvivar) [inline MyBus/MyBus/AppDelegate.swift:26]: Yes, we are to set Mapbox's token by code in the future in order to obfuscate it.

**Comment by lisandro** [comment]
Cool! First steps with Mapbox

**Comment by astradajulian** [comment]
Good way to start 👍

**Pull request merged**